seed/challenges/basic-javascript.json

```diff
@@ -911,21 +914,21 @@
         "For loops don't have to iterate one at a time. By changing our <code>final-expression</code>, we can count by even numbers.",
         "We'll start at <code>i = 0</code> and loop while <code>i < 10</code>.  We'll increment <code>i</code> by 2 each loop with <code>i += 2</code>.",
         "<code>var ourArray = [];</code>",
-        "<code>for(var i = 0; i < 10; i += 2) {</code>",
+        "<code>for (var i = 0; i < 10; i += 2) {</code>",
```

Comment:
I've never seen spaces after `for`.  I guess it's valid code, but seems unusual to me.